assets/src/edit-story/components/panels/stylePreset/test/panel.js

```diff
@@ -155,9 +153,7 @@ describe('Panels/StylePreset', () => {
           {
             id: '1',
             type: 'text',
-            color: [TEST_COLOR_2],
-            backgroundTextMode: BACKGROUND_TEXT_MODE.NONE,
-            font: TEXT_ELEMENT_DEFAULT_FONT,
+            content: '<span style="color: rgba(2, 2, 2)">Content</span>',
```

Comment:
Q: Why remove the other props, the `font` and `backgroundTextMode`? That's basically for ensuring that even though those exist, the logic should still consider only the color for the preset -- default font and `BACKGROUND_TEXT_MODE.NONE` should be ignored.

Reply:
Re-added in 16dbf27f.